lib/libshare/os/linux/nfs.c

```diff
@@ -634,114 +602,33 @@ nfs_update_shareopts(sa_share_impl_t impl_share, const char *resource,
 static void
 nfs_clear_shareopts(sa_share_impl_t impl_share)
 {
-	free(FSINFO(impl_share, nfs_fstype)->shareopts);
 	FSINFO(impl_share, nfs_fstype)->shareopts = NULL;
 }
 
 static const sa_share_ops_t nfs_shareops = {
 	.enable_share = nfs_enable_share,
 	.disable_share = nfs_disable_share,
+	.is_shared = nfs_is_share_active,
 
 	.validate_shareopts = nfs_validate_shareopts,
 	.update_shareopts = nfs_update_shareopts,
 	.clear_shareopts = nfs_clear_shareopts,
+	.commit_shares = nfs_export_all,
```

Comment:
rename the function to match the op name
```suggestion
	.commit_shares = nfs_commit_shares,
```